src/app/screens/Library/index.tsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import { View } from 'react-native';
+import BookList from '@app/components/BookList';
+import { BOOK_LIST } from '@constants/books';
+
+import styles from './styles';
+
+function Library() {
+  return (
+    <View style={styles.container}>
+      <BookList data={BOOK_LIST} />
+    </View>
+  );
```

Comment:
is this view necessary? Just asking

Reply:
I checked and it's necessary. It's so the booklist renders below the header.